subsystems/provisioning/provisioning.go

```diff
@@ -165,7 +165,7 @@ func (w *Provisioning) init(ctx context.Context) error {
 
 	w.checkConfigured()
 	if err := w.networkScan(ctx); err != nil {
-		return err
+		w.logger.Error(err)
```

Comment:
Some hardware can have (transient) errors during scan. (Log line looks like: "Scanning not allowed while unavailable") Michael Lee hit this and filed https://viam.atlassian.net/browse/RSDK-9252

Normally, not horrible, but during early startup provisioning (e.g. first boot for a machine without a config) it would cause permanent deadlock, since the normal monitoring loop is not yet running.

Reply:
The same "it's probably transient" issue applies to checkConnections() where a similar change as made.

Reply:
ohh got it and makes sense